opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -1765,14 +1771,22 @@ int Well::vfp_table_number() const {
 }
 
 double Well::inj_temperature() const {
-    if (this->wtype.injector() && this->well_inj_temperature)
-        return *this->well_inj_temperature;
+    if (!this->wtype.injector())
+        throw std::logic_error(fmt::format("Well {}: Cannot ask for injection temperature for a non-injector", this->name()));
+
+    if (!this->well_inj_temperature) {
+        if (this->default_well_inj_temperature) {
+            OpmLog::warning(fmt::format("Well {}: Injection temperature not specified, using default value of {}", this->name(), *this->default_well_inj_temperature));
+            return *this->default_well_inj_temperature;
```

Comment:
This warning is given every time inj_temperature() is called. As a result the .PRT file will be full of these warnings. Is that intended?

Reply:
Yes, I would like to give a good push to explicitly set the temperature (a simple WTEMP '*' would suffice), without actually failing if forgotten.